Clean up Vale errors in task topics (#922)
grafana/alloy: docs/sources/reference/components/loki.write.md

### endpoint block

The `endpoint` block describes a single location to send logs to.

> Multiple `endpoint` blocks can be provided to send logs to multiple locations.

Author: ```suggestion
You can use multiple `endpoint` blocks to send logs to multiple locations.
```

### queue_config block (experimental)

The optional `queue_config` block configures, when WAL is enabled, how the underlying client queues batches of logs sent to Loki.

> Refer to [Write-Ahead block](#wal-block-experimental).

Author: ```suggestion
Refer to [Write-Ahead block](#wal-block-experimental) for more information.
```
